backend/src/main/backend/Table.java

```diff
@@ -0,0 +1,100 @@
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+import java.util.ArrayList;
+
+public class Table {
+    private String name;
+
+    private Connection mConnection;
+    private PreparedStatement mCreateTable;
+    private PreparedStatement mDropTable;
+    private PreparedStatement mDeleteRow;
+    private PreparedStatement mInsertRow;
+    private PreparedStatement mUpdateRow;
+    // Other prepared statements...
+
+    public Table(String tName){
+        name = tName;
+    }
+
+    public String getName(){
+        return name;
+    }
+
+    public void create(Database db) {
```

Comment:
This method signature indicates to me that we plan to dynamically apply this statement, as well as the others, to multiple different databases rather than a single pre-determined database.

As we are only managing a single database, I would expect this method signature to be simply `create()` on a class instantiated with a reference to the proper database.